src/riscv/lib/src/jit/state_access.rs

```diff
@@ -153,19 +154,18 @@ register_jsa_functions!(
 extern "C" fn fregister_read<MC: MemoryConfig>(
     core: &mut MachineCoreState<MC, Owned>,
     reg: FRegister,
-) -> f64 {
+) -> FValue {
     let fval = core.hart.fregisters.read(reg);
```

Comment:
isn't the conversion stuff redundant? could we not just have 
```suggestion
) -> FValue {
    core.hart.fregisters.read(reg)
```

Reply:
Fixed [here](https://github.com/tezos/riscv-pvm/compare/553b46fd494e6e974eb87e3c4ad5d50c13a859dd..da72c2a29c6a2272d6f791c1f5434372bfd12e3b#diff-3efb6e3ea99258cc4124ee9f32f9d516d38f427d2f053d300a10f3097358182eL158-L159)